PullToRefreshDemo/ViewController.swift

```diff
@@ -14,16 +14,16 @@ class ViewController: UIViewController {
     @IBOutlet
     private var tableView: UITableView!
     
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        NSNotificationCenter.defaultCenter().addObserver(self, selector: "deviceRotated", name: UIDeviceOrientationDidChangeNotification, object: nil)
```

Comment:
In your fix it's better to override method viewWillTransitionToSize:withTransitionCoordinator rather than using NSNotificationCenter.